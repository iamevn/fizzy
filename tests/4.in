3,4:over
2,3:lap
2,6
