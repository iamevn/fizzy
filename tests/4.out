lap
overlap
lapover
5
lapover
